src/FINDOLOGIC/Export/CSV/CSVExporter.php

```diff
@@ -54,7 +54,8 @@ public function serializeItemsToFile(
         int $total = 0
     ): string {
         $csvString = $this->serializeItems($items, $start, $count, $total);
-        $targetPath = sprintf('%s/findologic.csv', $targetDirectory);
+
+        $targetPath = $this->getTargetPath($targetDirectory);
```

Comment:
:thinking: Something feels wrong about (ab)using `$targetDirectory` implicitly to contain a file path rather than a directory path. Is this really the job of *libflexport*?

If anything like this should even be possible at all, wouldn't it make more sense to make the file name configurable, rather than giving a parameter with a name that doesn't imply anything like that some magic semantics.

Reply:
Added a new `setFileName` method, which allows to alter the file name.